tests/unit/assessment/test_assessment.py

```diff
@@ -435,7 +435,7 @@ def test_cluster_assessment_cluster_policy_not_found(mocker):
         )
     ]
     ws = Mock()
-    ws.cluster_policies.get.side_effect = DatabricksError(error="NO_POLICY", error_code="NO_POLICY")
```

Comment:
wrong error code was checked. If the policy does not exists, you will get RESOURCE_DOES_NOT_EXIST